src/Tribe/Plugins.php

```diff
@@ -121,7 +121,14 @@ public function get_plugin_by_class( $main_class ) {
 		 * @return array
 		 */
 		public function get_list() {
-			return $this->tribe_plugins;
+			/**
+			 * Gives an opportunity to filter the list of tribe plugins
+			 *
+			 * @since TBD
+			 *
+			 * @return array
```

Comment:
This should be a `@param` tag (we're describing what's available to callbacks in this docblock, not what the method returns).